client/components/ModulesTreemap.jsx

```diff
@@ -208,6 +208,11 @@ export default class ModulesTreemap extends Component {
 
   handleConcatenatedModulesContentToggle = flag => {
     store.showConcatenatedModulesContent = flag;
+    if (flag) {
+      localStorage.setItem('showConcatenatedModulesContent', '1');
```

Comment:
Becomes this after implementing suggestions below:
```suggestion
      localStorage.setItem('showConcatenatedModulesContent', true);
```